src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -520,6 +520,21 @@ export default class AircraftModel {
         return this.positionModel.relativePosition;
     }
 
+    /**
+     * Returns the appropriate string to use in
+     * row one of a data block.
+     *
+     * @property timeShareDestination
+     * @type {string}
+     */
+    get timeShareDestination() {
```

Comment:
This element of the timeshare is actually called the `scratchpad`. It defaults to showing the values you've provided, but can also be used by the controller to input their own value, within the character and length limitations. The scratchpad is already plugged in and more or less set up to do exactly what this function does anyway, if you could just merge the two together.

Also, we are missing a `.toUpperCase()` when the destination airport is being carried into the data block.

Reply:
The only references I'm seeing to `scratchpad` as an existing concept are within the changelog and a breif item in the command reference doc about it not being implemented?

`.toUpperCase()` is done

Reply:
`RadarTargetModel._scratchPadText`. It's currently already being initialized to `aircraftModel.destination.substr(1)` by the RadarTargetModel. And it can be changed (including by the user) in `RadarTargetModel.setScratchpad()`.

We should use the scratchpad in lieu of this getter.